modules/ROOT/pages/backup-restore/copy-database.adoc

```diff
@@ -210,6 +210,18 @@ The block format is the default format for all newly-created databases as long a
 For more information on the block format, see xref:database-internals/store-formats.adoc[Store formats].
 ====
 
+[NOTE]
+====
+In Neo4j 2025.01, a new dual-named option `--from-pagecache, --max-off-heap-memory=<size>` replaces the `--from-pagecache=<size>` option, which was used to specify how much cache to allocate when reading the source.
+
+The new option determines the maximum amount of off-heap memory available during the copy process for reading and writing.
+This change helps you to control off-heap memory rather than just allocate cache.
+
+To configure the off-heap memory value, you can use either `--from-pagecache=<size>` or `--max-off-heap-memory=<size>`.
+Note that the `--from-pagecache` option is considered deprecated and may be removed in future versions.
```

Comment:
Is it deprecated or not?

Reply:
I guess the old name `--from-pagecache` will be removed in the future, so I suppose we have to say about it. If the word 'deprecated' is too confusing, we need to rephrase this sentence. 
@tinwelint, any comments?